core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1775,12 +1776,21 @@ public String getFullDisplayName() {
      */
     @Exported(name="jobs")
     public List<TopLevelItem> getItems() {
+        return getItems(t->true);
+    }
+
+    /**
+     * Gets just the immediate children of {@link Jenkins} based on supplied predicate.
```

Comment:
```suggestion
     * Gets just the immediate child of {@link Jenkins} based on supplied predicate.
```

Reply:
Since its a list of items it should be children instead of child

Reply:
Ah, right!